wasp/apps/template.py

```diff
@@ -44,23 +44,30 @@ def __init__(self):
         pass
 
     def foreground(self):
-        """Activate the application."""
+        """Called when the application is launched, for example when the user
+        presses its icon in the launcher."""
         self._draw()
         wasp.system.request_event(wasp.EventMask.TOUCH |
                                   wasp.EventMask.SWIPE_UPDOWN |
                                   wasp.EventMask.BUTTON)
         wasp.system.request_tick(1000)
 
     def background(self):
-        """De-activate the application."""
+        """Called when the leaving the application, either because of screen
+        timeout of directly by the user."""
         pass
 
     def sleep(self):
-        """Notify the application the device is about to sleep."""
+        """Called by the system to notify the application that the screen is
+        about to be turned off. Most apps should not implement it.
```

Comment:
Here I want to keep the "go to sleep" to be preserved. Although "go to sleep and turn off the screen" might be OK.